### ISSUE TITLE
Is there a way to get the underlying version of the ngrok executable?

### ISSUE THREAD

**Issue by philnash** [bubenshchykov/ngrok#177]
And if so, is it also possible to find out if there is a new version available?

I know that the ngrok executable itself knows (or can know) both of these things. Just wondering if that can be exposed as part of this module?

Thanks!

**Comment by bubenshchykov** [comment]
should be possible, smth like
```
const util = require('util');
const exec = util.promisify(require('child_process').exec);

async function getVersion() {
  try {
    const { stdout, stderr } = await exec('ngrok --version'); // todo: pass correct path to the bin
    return stdout; // extract version if there is extra output
  } catch (e) {
   
  }
}
```
please feel free to add it!

not sure if ngrok binary has command for checking the latest version though

**Comment by philnash** [comment]
Awesome, thanks, I will take a look at adding it.

Just to let you know, this project is underpinning my [VSCode extension for ngrok](https://github.com/philnash/ngrok-for-vscode/) which is working very nicely. Thanks for your work on this!

**Comment by bubenshchykov** [comment]
The extension looks super awesome, I will definitely give it a try!

```
console.log("🍻")
```

**Comment by bubenshchykov** [comment]
btw, you should ping @inconshreveable and list it on https://ngrok.com/docs#compat-subject#visual-studio

**Comment by philnash** [comment]
This has been implemented and merged in #181.
